packages/wiz/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.3"
```

```suggestion
- version: "1.8.0"
```
(enhancements bump minor)